### ISSUE TITLE
NTR: Ammonia: no rise with ischemic exercise test (Associated motor)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7775]
New term request Ammonia: no rise with ischemic exercise test (segal_070319180858):

**Comment by pnrobinson** [comment]
@MickeySegal
It seems that
Failure to elevate lactate with ischemic exercise test
is the "main" feature with 
Failure to elevate ammonia with ischemic exercise test
being a confirmatory observation
--
https://pubmed.ncbi.nlm.nih.gov/3476851/

Is this what you are referring to and should we also add the lactate term?

**Comment by MickeySegal** [comment]
We do have the finding "Lactate: no rise with ischemic exercise test" as well, with nativeCode f301

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000320
- label: Failure to elevate ammonia on ischemic exercise
- parents: [Abnormal cardiovascular system physiology (HP:0011025)](https://hpo.jax.org/app/browse/term/HP:0011025)
- pmid: PMID:3476851
- def: Normal subjects display an increase in the concentration of ammonia in the blood following an ischemic exercise test. This term refers to an increase in ammonia that is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7775
- synonyms:
